coderwassananmol/BUB2
# MicroTask T315134. Add search bar in queue

**Issue**: T315134.  Add search bar in queue

**Proposed Changes**
- Implement real-time search functionality.
  - The search will happen automatically as the user types in the search bar.
  - The search will be based on both job ID and title.
  - Placeholder text in the search bar should be set to "Search by job ID or title" to guide the user.
- Place the search bar to the top of the table for improved user context, making it clear that the search is table-specific.

**Testing**
- Make sure the search bar works in real-time, updating the table as the user types.
- Test that the search function is working correctly for both job ID and title.
- Verify that the placeholder text in the search bar displays as "Search by job ID or title."
- Check that the search bar is positioned at the top of the table.

<img width="1576" alt="Screenshot 2023-10-11 at 07 00 43" src="https://github.com/coderwassananmol/BUB2/assets/33901311/4e70ea8a-7cda-4ffc-bcfd-d7c2a97b2d79">
